src/styles/main.scss

```diff
@@ -1,3 +1,257 @@
-body {
+$blue-accent: #00acdc;
+$main-accent: #060b35;
+$secondary-accent: #060b35;
+$white: #fff;
+
+$cardAmount: 4;
+$cardWith: 200px;
+$cardMargin: 48px;
+
+html {
+  font-family: roboto, Arial, sans-serif;
+}
+
+body,
+h2,
+p {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background: $white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
```

Comment:
don't forget about css properties order
http://codeguide.academy/html-css.html#css-order

Reply:
also you should split your bem blocks (checklist)
<img width="651" alt="image" src="https://user-images.githubusercontent.com/102411130/210745500-61a34ad1-0603-4258-9b40-8531b8a860be.png">